components/Board.tsx

```diff
@@ -24,9 +24,12 @@ const GridContainer = styled.div`
   flex-direction: column;
 `;
 
-const BoardTitle = styled.h3`
+const BoardTitle = styled.div`
```

Comment:
I played with the styling a bit. Instead of giving the clear icon a margin I used `align-items: center` to have it be centered. Also, changed the h3 margin a bit. Once you changed BoardTitle to be a div, the h3 got some default margins and it's a bit too much

![image](https://user-images.githubusercontent.com/2545447/117527339-e92e0680-afd3-11eb-8490-38b860d26ab2.png)

Reply:
ok good I'll take a look,